### ISSUE TITLE
Cannot see speakers and sessions data

### ISSUE THREAD

**Issue by larrywa** [microsoft/MCW-Cloud-native-applications#268]
I've gone through the steps twice. Everything seems to work ok except that when I go to the Contoso site, and click on Sessions or Speakers link, I see no data. The data is in CosmosDB. There are not errors.

Any ideas what to look at? Even though I've gone through it twice there must be some small detail I'm missing.

Thanks!

**Comment by hatboyzero** [comment]
Can you provide any relevant log output from your browser debug console when browsing to the Sessions and/or Speakers link?

**Comment by timahenning** [comment]
Hi @larrywa, can you provide the log output?

**Comment by larrywa** [comment]
Sorry for the delay.

I ended up having to tear the whole environment down due to costs. Most of the students I had did not seem to have the same issue so I'll assume it is some step I missed.

Most of the issues we had were will people getting lost in all the steps going back and forth to Github and then local repositories etc. It all worked but they couldn't take a break in the middle of the process, otherwise they would forget where they were.

Thanks!

larrywa
